index.js

```diff
@@ -1,5 +1,18 @@
 "use strict";
 var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
 fluid.module.register("gpii-pouchdb", __dirname, require);
 
-require("./src/js/pouch");
\ No newline at end of file
+require("./src/js/pouch");
+
+fluid.registerNamespace("gpii.pouch");
+
+gpii.pouch.loadTestingSupport = function () {
+    // require("gpii-express");
```

Comment:
I can't reproduce any error from uncommenting these definitions - can you confirm + paste?